core/src/main/resources/config-store/config-cmis.xml

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (C) 2001-2016 Food and Agriculture Organization of the
+  ~ United Nations (FAO-UN), United Nations World Food Programme (WFP)
+  ~ and United Nations Environment Programme (UNEP)
+  ~
+  ~ This program is free software; you can redistribute it and/or modify
+  ~ it under the terms of the GNU General Public License as published by
+  ~ the Free Software Foundation; either version 2 of the License, or (at
+  ~ your option) any later version.
+  ~
+  ~ This program is distributed in the hope that it will be useful, but
+  ~ WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+  ~ General Public License for more details.
+  ~
+  ~ You should have received a copy of the GNU General Public License
+  ~ along with this program; if not, write to the Free Software
+  ~ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
+  ~
+  ~ Contact: Jeroen Ticheler - FAO - Viale delle Terme di Caracalla 2,
+  ~ Rome - Italy. email: geonetwork@osgeo.org
+  -->
+
+<beans
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns="http://www.springframework.org/schema/beans"
+        xsi:schemaLocation="
+		http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans.xsd
+    ">
+
+<!--   If you want data stored in cmis compliant application, use the following: -->
+<!-- Alfrsco sample -->
+<!--    <bean id="cmisconfiguration" class="org.fao.geonet.resources.CMISConfiguration">-->
+<!--      <property name="baseRepositoryPath" value="geonetwork"/>-->
+<!--      <property name="servicesBaseUrl" value="http://localhost:8080/alfresco"/>-->
+<!--      <property name="bindingType" value="browser"/>-->
+<!--      <property name="browserUrl" value="/api/-default-/public/cmis/versions/1.1/browser"/>-->
+<!--      <property name="username" value="username"/>-->
+<!--      <property name="password" value="password"/>      -->
+<!--      &lt;!&ndash;property name="bindingType" value="atompub"/>-->
+<!--      <property name="atompubUrl" value="/api/-default-/public/cmis/versions/1.1/atom"/&ndash;&gt;-->
+<!--      &lt;!&ndash;property name="bindingType" value="webservices"/>-->
+<!--      <property name="webservicesBaseUrl" value="/cmisws"/&ndash;&gt;-->
+<!--    </bean>-->
+
+<!-- Open Text sample -->
+<!--<bean id="cmisconfiguration" class="org.fao.geonet.resources.CMISConfiguration">-->
+<!--    <property name="repositoryName" value="Enterprise"/>-->
```

Comment:
Open text has several repositories so we had to specify a repository name.  The default main one seems to be Enterprise.
Alfresco only has one repository.  So if it was not supplied, it was picking the first one.  I checked again and I noticed that the default repository has repository id = `-default-`
I also realized that it may be a bad practice to pick the first repository.  If a second is ever supported then it could end up picking the wrong repository on startup.  So I removed the logic to pick the first repo and changed the example so that it uses  `CMIS_REPOSITORY_ID=-default-`